rotkehlchen/api/v1/schemas.py

```diff
@@ -1577,6 +1577,11 @@ class ExchangesDataResourceSchema(Schema):
     location = LocationField(limit_to=ALL_SUPPORTED_EXCHANGES, load_default=None)
 
 
+class ExchangeEventsQuerySchema(AsyncQueryArgumentSchema):
+    name = fields.String()
```

Comment:
**Always** specify these stuff. Do not rely on default arguments which may change one release to the next.

```suggestion
    name = fields.String(required=False)
```